pkgs/applications/audio/spotify-player/default.nix

```diff
@@ -7,6 +7,15 @@
 , alsa-lib
 , dbus
 , fontconfig
+
+# build options
+, withStreaming ? true
+, withAudioBackend ? "rodio"
+, withMediaControl ? true
+, withLyrics ? false
+, withImage ? false
+, withNotify ? false
+, withSixel ? false
```

Comment:
Why not build with all features by default?

Reply:
Yeah, I don't know why the maintainer didn't merge the PR. PR now needs to be updated for all the new features of spotify_player, but other than that pretty much nothing has changed. 

Having reasonable defaults, or even setting everything as default does not affect the average consumer at all, it just allows others to override when necessary. Pretty basic feature of flakes, and is the standard for packages with compilation options. Potentially options could be disabled at runtime with config in rust and that is the reason to compile with everything? But still, people then have to deal with home-manager or something like that, and it also means that they have an inflated binary.